cli/src/program.rs

```diff
@@ -1728,10 +1728,7 @@ fn process_set_authority(
             .send_and_confirm_transaction_with_spinner_and_config(
```

Comment:
I can't comment higher up, but there's a `send_and_confirm_transaction_with_spinner` call on line 1578 that isn't using the send transaction config -- could you add it there?

Reply:
Oh thanks! I missed that